src/containers/VisibleMovie/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+
+import MovieEditForm from '../../components/Modal/EditMovieForm';
+import { editMovie, movieByIdSelector } from '../../slices/moviesInfo';
+
+const VisibleMovieForm = ({ movieId }) => {
+  const dispatch = useDispatch();
+  const movie = useSelector((state) => movieByIdSelector(state, movieId));
```

Comment:
It's better to use the API here to receive movie details as usually edit form is available using URL

Reply:
done